crates/wash-cli/src/bin/wash.rs

```diff
@@ -120,6 +121,13 @@ struct Cli {
     )]
     pub(crate) experimental: bool,
 
+    #[clap(
+        long = "markdown-help", 
+        hide = true,
+        global = true
+    )]
+    markdown_help: bool,
```

Comment:
What do you think about calling this something like `--help-output-markdown` and maybe mark it as [required with `--help`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.requires) ? 

That *should* force usage like this:

```console
wash app list --help --help-output-markdown
```

I'm also definitely open to `--help-markdown` and [maybe setting `conflicts_with`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with), if the above is too verbose!

Reply:
It looks like it gives incorrect output if you run the argument with the --help argument, so I took the `conflicts_with` approach with the most recent commit (and changed the argument to `--help-markdown`).

Reply:
Ah thanks for getting to this!